### PR TITLE
Update FlightsPage.vue

### PR DESCRIPTION
**Pull request by paulaasensio**

### CODE REVIEW

**Comment by Mexaby** [comment]
Looks great! Make sure to add checks and warnings, for instance if the return date is before the departure date, and add some sort of results as well. It can be just a table that shows the travel company, departure and arrival hours, length of the trip, seats available, etc.

Now that I think of it, perhaps you could rename this page to "Travel" and add another dropdown menu to select the means of travel (bus, car, plane, train, boat?) and have results based on that.

**Comment by paulaasensio** [comment]
I have finish the travel template with the new dropdown and the conditions and error!

**Comment by Mexaby** [comment]
- **CRITICAL** - same destination error is displayed even is the destinations are different
image
- the destination results overlap the search button (move the search button to the right of the inputs)
image
- no error displayed if means of transport dropdown is left empty (perhaps add an "any" option that displays all options in the list at once)


**The critical bug needs to be fixed ASAP! Can't progress testing it further.**

File: travel-assistant/src/components/ListTravels.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<template>
```
Comment (Mexaby) [inline travel-assistant/src/components/ListTravels.vue:33]: What is the point of this component? You're not using it anywhere

**Review by Mexaby — APPROVED** [review]
Good work!

**Pull request merged**